store/migration/dev/20230106000000##change_migration_history_id_type_to_string.sql

```diff
@@ -0,0 +1,8 @@
+UPDATE task
+SET payload = payload || jsonb_build_object('migrationID', payload->>'migrationID'::TEXT)
```

Comment:
If ThreadID and MigrationID is only used in TaskDatabaseDataUpdatePayload https://sourcegraph.com/github.com/bytebase/bytebase/-/blob/api/task.go?L182&subtree=true

I would suggest to ignore it since it's not used in production yet, also let's correct the annotation to threadId and migrationId